setup.py

```diff
@@ -1,87 +1,39 @@
-    formatter = logging.Formatter(
-        '%(asctime)s - %(name)s - %(levelname)s - %(message)s')
-    # add formatter to ch
-    ch.setFormatter(formatter)
-    # add ch to logger
-    log.addHandler(ch)
-
-
-INSTALL_REQUIRES = [
-    'six',
-    "SQLAlchemy>=1.1.0",
-    "futures>=3.1.1",
-]
-
-setup(
-    name=dist_name,
-    version=__version__,
-    description=description,
-    long_description="{readme}".format(readme=readme),
-    author='Learning Equality',
-    author_email='aron+barbequeue@learningequality.org',
-    url='https://github.com/learningequality/barbequeue',
-    packages=find_packages('src'),
-    package_dir={'': 'src'},
-    include_package_data=True,
-    license='MIT',
-    zip_safe=False,
-    install_requires=INSTALL_REQUIRES,
-    keywords=('queue', 'async'),
-    classifiers=[
-        'Development Status :: 2 - Pre-Alpha',
-        'Intended Audience :: Developers',
-        'License :: OSI Approved :: MIT License',
-        'Natural Language :: English',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
-        'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6',
-        'Programming Language :: Python :: Implementation :: PyPy',
-    ], )
+# DO NOT EDIT THIS FILE -- AUTOGENERATED BY PANTS
+# Target: PythonLibrary(BuildFileAddress(src/BUILD, lib))
+
+from setuptools import setup
+
+setup(**
+{   'author': 'Learning Equality',
+    'author_email': 'aron@learningequality.org',
+    'classifiers': [   'Development Status :: 2 - Pre-Alpha',
+                       'Intended Audience :: Developers',
+                       'License :: OSI Approved :: MIT License',
+                       'Natural Language :: English',
+                       'Programming Language :: Python :: 2',
+                       'Programming Language :: Python :: 2.7',
+                       'Programming Language :: Python :: 3',
+                       'Programming Language :: Python :: 3.5',
+                       'Programming Language :: Python :: 3.6'],
+    'description': '',
+    'install_requires': ['SQLAlchemy>=1.1.10', 'futures>=3.1.1'],
+    'keywords': ('queue', 'async'),
+    'license': 'MIT',
+    'long_description': '',
+    'name': 'iceqube',
+    'package_data': {   },
+    'package_dir': {   '': 'src'},
+    'packages': [   'iceqube',
+                    'iceqube.common',
+                    'iceqube.messaging',
+                    'iceqube.messaging.backends',
+                    'iceqube.scheduler',
+                    'iceqube.storage',
+                    'iceqube.storage.backends',
+                    'iceqube.worker',
+                    'iceqube.worker.backends'],
+    'url': 'https://github.com/learningequality/iceqube',
+    'version': '0.0.1',
+    'zip_safe': True}
```

Comment:
This can be harmful if you package has data files and they aren't accessed with `find_package_data` (or whatever it's called). Often people set this to `False`, having it `True` means that the package may sit as an Egg on the target system and run dynamically.